internal/pkg/cli/svc_deploy.go

```diff
@@ -444,6 +457,20 @@ func (o *deploySvcOpts) deploySvc(addonsURL string) error {
 	return nil
 }
 
+func (o *deploySvcOpts) validateAppVersion(svc *manifest.LoadBalancedWebService) error {
+	appVersion, err := o.appVersionGetter.Version()
```

Comment:
Do we still need to get app version if `aws.StringValue(svc.Alias) == "" || !o.targetApp.RequiresDNSDelegation()` ?If not, I wonder if we should only get version when the condition applies? Because it could be the case where `.Version()` returns an error, and thus stopping the execution, while we don't really need that piece of information in the first place.